tools/getting-started/src/run-scripts.tsx

```diff
@@ -23,6 +25,13 @@ const RunScripts: React.FC<Props> = props => {
   const [isRenamingProjectValid, setIsRenamingProjectValid] = useState<boolean>()
   const [renameProjectOutput, setRenameProjectOutput] = useState("")
 
+  const [isDeployingApplication, setIsDeployingApplication] = useState(false)
+  const [isDeployingApplicationValid, setIsDeployingApplicationValid] = useState(
+    !!props.digitalOceanToken ? undefined : true
```

Comment:
why to we need it to be set to undefined instead of false?

Reply:
If it were set to false, it would have the `X` icon in red. Undefined doesn't display anything.